lib/iris/experimental/ugrid.py

```diff
@@ -5,42 +5,632 @@
 # licensing details.
 
 """
-Ugrid functions.
+Infra-structure for unstructured mesh support, based on
+CF UGRID Convetions (v1.0), https://ugrid-conventions.github.io/ugrid-conventions/
 
 """
 
-import iris
+from functools import wraps
 
+import dask.array as da
+import numpy as np
 
-def ugrid(location, name):
+from .. import _lazy_data as _lazy
+from ..common.metadata import (
+    BaseMetadata,
+    metadata_manager_factory,
+    SERVICES,
+    SERVICES_COMBINE,
+    SERVICES_EQUAL,
+    SERVICES_DIFFERENCE,
+)
+from ..common.lenient import _lenient_service as lenient_service
+from ..config import get_logger
+from ..coords import _DimensionalMetadata
+
+
+__all__ = [
+    "Connectivity",
+    "ConnectivityMetadata",
+]
+
+
+# Configure the logger.
+logger = get_logger(__name__, fmt="[%(cls)s.%(funcName)s]")
```

Comment:
I'm not clear why this is necessary? `logger` isn't used here. Perhaps it's used by the parent classes, but in that case I assume we wouldn't need to import it here too?

Reply:
It's interesting that `flake8` doesn't complain about this... I though that it might.

Happy to remove it :+1: